src/state/wallet/hooks.ts

```diff
@@ -65,12 +66,27 @@ export function useETHBalances(
  */
 export function useTokenBalancesWithLoadingIndicator(
   address?: string,
-  tokens?: (Token | undefined)[],
+  tokens?: (
+    | {
+        chainId: ChainId;
+        address: string;
+        decimals: number;
+        symbol?: string;
+        name?: string;
+      }
+    | undefined
+  )[],
 ): [{ [tokenAddress: string]: TokenAmount | undefined }, boolean] {
-  const validatedTokens: Token[] = useMemo(
+  const validatedTokens: any[] = useMemo(
```

Comment:
We need to add typing back in the future, probably just have to upgrade v2 to use the new sdk in generally but thats a big project